test/test_consumer_group.py

```diff
@@ -29,6 +29,15 @@ def test_consumer(kafka_broker, topic, version):
     assert consumer._client._conns[node_id].state is ConnectionStates.CONNECTED
     consumer.close()
 
+@pytest.mark.skipif(not version(), reason="No KAFKA_VERSION set")
+def test_consumer_topics(kafka_broker, topic, version):
+    consumer = KafkaConsumer(bootstrap_servers=get_connect_str(kafka_broker))
+    consumer.subscribe([topic])
```

Comment:
Is subscribe() needed? Couldn't you just call topics() and partitions_for_topic() since they are both blocking for metadata? You may need to `poll()` at some point (even without an active subscription) in order to drive the I/O.

Reply:
Let me try that -- I do think it'd be good to add on other tests eventually that would check subscribe() and other methods

Reply:
@jeffwidman I changed the test to just have poll and removed subscribe

Reply:
> I do think it'd be good to add on other tests eventually that would check subscribe() and other methods

Absolutely agree. Happily welcome a PR. Just rather keep those in a separate test.